src/main/java/in/projecteka/consentmanager/consent/ConsentArtefactsController.java

```diff
@@ -32,7 +34,7 @@
         return consentManager.getConsentArtefactLight(consentId);
     }
 
-    @GetMapping(value = "/consent-requests/{request-id}/consent-artefacts")
+    @GetMapping(value = "/"+CA_SCHEMA_VERSION+"/consent-requests/{request-id}/consent-artefacts")
```

Comment:
URL versioning need to be done across all the URLs, it can be done later.

Reply:
I will revert these.